nemo/collections/nlp/modules/common/text_generation_strategy.py

```diff
@@ -132,6 +134,36 @@ def post_process(self, tokens: torch.Tensor, new_tokens: torch.Tensor, context_l
         """
         pass
 
+    def end_of_generation_condition(
+        self, tokens: torch.Tensor, prev: torch.Tensor, eod_id: int, end_strings: List[str]
+    ) -> torch.Tensor:
+        """
+        return whether the generation should stop based on the previous token
+        Args:
+            tokens (torch.Tensor): the generated tokens so far
+            prev  (torch.Tensor): the previous token
+            eod_id (int): the end of document token id
+            end_strings (List[str]): the list of end of generation strings
+        returns:
+            a boolean tensor indicating whether the generation should stop
+        """
+        if len(end_strings) == 1 and end_strings[0] == END_OF_SEQ:
```

Comment:
Can you add to comment near definition of `END_OF_SEQ`it is the default `<EOS>`?

Reply:
yes. will add the comment.